test/shared_test/lib_battery_lifetime_test.cpp

```diff
@@ -338,15 +337,91 @@ TEST_F(lib_battery_lifetime_test, runCycleLifetimeTestWithRestPeriod) {
         idx++;
     }
 
-    
-    cycle_state s = *model->get_state().cycle;
-    EXPECT_NEAR(s.q_relative_cycle, 99.96, tol);
-    EXPECT_NEAR(s.rainflow_Xlt, 90, tol);
-    EXPECT_NEAR(s.rainflow_Ylt, 90, tol);
-    EXPECT_NEAR(s.rainflow_jlt, 2, tol);
+
+    lifetime_state s = model->get_state();
+    EXPECT_NEAR(s.cycle->q_relative_cycle, 99.96, tol);
+    EXPECT_NEAR(s.cycle->rainflow_Xlt, 90, tol);
+    EXPECT_NEAR(s.cycle->rainflow_Ylt, 90, tol);
+    EXPECT_NEAR(s.cycle->rainflow_jlt, 2, tol);
     EXPECT_NEAR(s.range, 90, tol);
     EXPECT_NEAR(s.average_range, 90, tol);
     EXPECT_NEAR(s.n_cycles, 2, tol);
 }
 
+TEST_F(lib_battery_lifetime_nmc_test, updateCapacityTest) {
```

Comment:
Thanks for these tests! It looks like these cover two out of 11 cells in the paper. Can you add the following?
- Cycling at different temperatures
- Storage at different temperatures
- Different C rates

Reply:
Added cycling and storage tests with different temperatures and C rates to lib_battery_lifetime_test.cpp-- the results are now tracking well with the static model from the paper.